qiskit_nature/__init__.py

```diff
@@ -26,9 +26,18 @@
    :toctree: ../stubs/
    :nosignatures:
 
+   QiskitNatureLogging
```

Comment:
I'm still not convinced whether this is the right way to document this..

As things are now, a user will modify the logging via `qiskit_nature.settings.logging`. However, the documentation may lead a user to believe that they can simply instantiate `QiskitNatureLogging()` and use that.

Maybe we should follow the same approach as the settings singleton here? I.e.
```python
# in qiskit_nature/logging.py
logging = QiskitNatureLogging()

# in qiskit_nature/__init__.py
from .logging import logging
# and then expose logging as part of Globals (just like settings)
```

Reply:
Changed